File: 프로그래머스/lv1/12950. 행렬의 덧셈/행렬의 덧셈.kt
class Solution {
    fun solution(arr1: Array<IntArray>, arr2: Array<IntArray>): Array<IntArray> {
        var answer = Array(arr1.size) { IntArray(arr1[0].size) }
        for(i in arr1.indices) {
            val arr = IntArray(arr1[i].size) { 0 }
            for(j in arr1[i].indices) {
                arr[j] = arr1[i][j] + arr2[i][j]
            }
            answer[i] = arr
        }
        return answer
    }
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.swift
func solution(_ arr1: [[Int]], _ arr2: [[Int]]) -> [[Int]] {
    var answer: [[Int]] = []
    
    for i in 0...arr1.count-1 {
        var arr: [Int] = []
        
        for j in 0...arr1[i].count-1 {
            arr.append(arr1[i][j] + arr2[i][j])
        }
        answer.append(arr)
    }
    
    return answer
}